### PR TITLE
Point pinniped registry to ghcr.io

### PR DESCRIPTION
**Pull request by devacts**
#### What this PR does / why we need it:

Pinniped test cases fail in the kapp release job due to ImagePullBackOff Error
as projects.registry.vmware.com is no longer public.
This PR now points the pinniped image registry to ghcr.io
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

##### Review Checklist:

- [ ] Follows the [developer guidelines](https://carvel.dev/shared/docs/latest/development_guidelines/)
- [ ] Relevant tests are added or updated
- [ ] Relevant docs in this repo added or updated
- [ ] Relevant carvel.dev docs added or updated in a separate PR and there's
  a link to that PR
- [ ] Code is at least as readable and maintainable as it was before this
  change

#### Additional documentation e.g., Proposal, usage docs, etc.:

```docs

```

### CODE REVIEW

File: examples/pinniped-v0.13.0/config.yml [hunk 1]
```diff
@@ -549,7 +549,7 @@ spec:
       serviceAccountName: pinniped-concierge
       containers:
       - name: pinniped-concierge
-        image: projects.registry.vmware.com/pinniped/pinniped-server:v0.13.0@sha256:7bdd608100ef594d51cb36d63b87558524140c349cb2bf925338e8d7c11e208f
+        image: https://ghcr.io/vmware-tanzu/pinniped/pinniped-server:v0.32.0@sha256:f76fa757678f1ab2492be698dc33afbec5ce22b32eebb8a648d5196f9e63ce35
```
Comment (rohitagg2020) [inline examples/pinniped-v0.13.0/config.yml]:
```suggestion
        image: ghcr.io/vmware-tanzu/pinniped/pinniped-server:v0.32.0@sha256:f76fa757678f1ab2492be698dc33afbec5ce22b32eebb8a648d5196f9e63ce35
```

File: examples/pinniped-v0.13.0/config.yml [hunk 2]
```diff
@@ -1172,7 +1172,7 @@ spec:
       serviceAccountName: local-user-authenticator
       containers:
       - name: local-user-authenticator
-        image: projects.registry.vmware.com/pinniped/pinniped-server:v0.13.0@sha256:7bdd608100ef594d51cb36d63b87558524140c349cb2bf925338e8d7c11e208f
+        image: https://ghcr.io/vmware-tanzu/pinniped/pinniped-server:v0.32.0@sha256:f76fa757678f1ab2492be698dc33afbec5ce22b32eebb8a648d5196f9e63ce35
```
Comment (rohitagg2020) [inline examples/pinniped-v0.13.0/config.yml]:
```suggestion
        image: ghcr.io/vmware-tanzu/pinniped/pinniped-server:v0.32.0@sha256:f76fa757678f1ab2492be698dc33afbec5ce22b32eebb8a648d5196f9e63ce35
```

File: examples/pinniped-v0.13.0/config.yml [hunk 3]
```diff
@@ -1406,7 +1406,7 @@ spec:
       serviceAccountName: pinniped-supervisor
       containers:
       - name: pinniped-supervisor
-        image: projects.registry.vmware.com/pinniped/pinniped-server:v0.13.0@sha256:7bdd608100ef594d51cb36d63b87558524140c349cb2bf925338e8d7c11e208f
+        image: https://ghcr.io/vmware-tanzu/pinniped/pinniped-server:v0.32.0@sha256:f76fa757678f1ab2492be698dc33afbec5ce22b32eebb8a648d5196f9e63ce35
```
Comment (rohitagg2020) [inline examples/pinniped-v0.13.0/config.yml]:
```suggestion
        image: ghcr.io/vmware-tanzu/pinniped/pinniped-server:v0.32.0@sha256:f76fa757678f1ab2492be698dc33afbec5ce22b32eebb8a648d5196f9e63ce35
```

File: examples/pinniped-v0.13.0/config.yml [hunk 4]
```diff
@@ -521,7 +521,7 @@ metadata:
...
...ay be set here, although other YAML references to the default port (10250) may also need to be updated\n# impersonationProxyServerPort may be set here, although other YAML references to the default port (8444) may also need to be updated\nnames:\n  servingCertificateSecret: pinniped-concierge-api-tls-serving-certificate\n  credentialIssuer: pinniped-concierge-config\n  apiService: pinniped-concierge-api\n  impersonationLoadBalancerService: pinniped-concierge-impersonation-proxy-load-balancer\n  impersonationClusterIPService: pinniped-concierge-impersonation-proxy-cluster-ip\n  impersonationTLSCertificateSecret: pinniped-concierge-impersonation-proxy-tls-serving-certificate\n  impersonationCACertificateSecret: pinniped-concierge-impersonation-proxy-ca-certificate\n  impersonationSignerSecret: pinniped-concierge-impersonation-proxy-signer-ca-certificate\n  agentServiceAccount: pinniped-concierge-kube-cert-agent\nlabels: {\"app\":\"pinniped-concierge\"}\nkubeCertAgent:\n  namePrefix: pinniped-concierge-kube-cert-agent-\n  \n  \n  image: https://ghcr.io/vmware-tanzu/pinniped/pinniped-server:v0.32.0@sha256:f76fa757678f1ab2492be698dc33afbec5ce22b32eebb8a648d5196f9e63ce35\n  \n  \n  \n\n"
```
Comment (rohitagg2020) [inline examples/pinniped-v0.13.0/config.yml]: remove `https:// ` from the image url.

**Review by rohitagg2020 — APPROVED** [review]

**Review by praveenrewar — APPROVED** [review]

**Pull request merged**